pkg/kubelet/kubeletconfig/checkpoint/store/fsstore_test.go

```diff
@@ -300,14 +300,13 @@ func TestFsStoreAssignedModified(t *testing.T) {
 	saveTestSourceFile(t, store, assignedFile, nil)
 
 	// set the timestamps to the current time, so we can compare to result of store.AssignedModified
-	now := time.Now()
+	now := time.Now().Round(time.Second)
 	err = store.fs.Chtimes(store.metaPath(assignedFile), now, now)
 	if err != nil {
 		t.Fatalf("could not change timestamps, error: %v", err)
 	}
 
-	// for now we hope that the system won't truncate the time to a less precise unit,
-	// if this test fails on certain systems that may be the reason.
+	// compare the times with less precise unit because certain systems truncate the time to a less precise unit.
```

Comment:
Can you move this comment to where we round the time?
Since it's not next to the comparison anymore, maybe reword to `round time to the nearest second because some file systems do not support sub-second precision`.